lib/commands/update.js

```diff
@@ -195,6 +197,24 @@ class UpdateCommand extends Command {
         });
     }
 
+    rollbackFromFail(error, newVer, force = false) {
+        const oldVer = this.system.getInstance().cliConfig.get('previous-version');
+        const question = `Unable to upgrade Ghost from v${oldVer} to v${newVer}. Would you like to revert back to v${oldVer}?`;
+        const promise = force ? Promise.resolve(true) : this.ui.confirm(question, true);
```

Comment:
We need to reorder the executions here, because the log output is broken. That is because we already trigger the confirm screen before we output the error. 

1. print error
1. if force, return this.run
2. run ui.confirm